doc/man3/ASN1_TIME_set.pod

```diff
@@ -102,8 +102,8 @@ functions check the syntax of the time structure I<s>.
 
 The ASN1_TIME_print(), ASN1_UTCTIME_print() and ASN1_GENERALIZEDTIME_print()
 functions print the time structure I<s> to BIO I<b> in human readable
-format. It will be of the format MMM DD HH:MM:SS YYYY [GMT], for example
-"Feb  3 00:55:52 2015 GMT", which does not include a newline.
+format. It will be of the format MMM DD HH:MM:SS YYYY[.s*] GMT, for example
```

Comment:
Should the fractional seconds follow `SS`?

Reply:
Of course. My bad :-(( Thanks for the catch! Corrected in latest commit.